db/schema.rb

```diff
@@ -12,15 +12,15 @@
 
 ActiveRecord::Schema.define(version: 2019_09_26_144109) do
 
-  create_table "application_drafts", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "application_drafts", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```

Comment:
I'm not sure what's going on here, or if it was intentional. If not intentional, are you sure you migrated/seeded your database correctly? To my understanding this file should only be affected by writing new migrations, which there aren't any.

Reply:
I think these changes might be OK. He merely ran migrations to get his tables and columns and it just happened. I believe it won't break anything for production.

Reply:
So @lacayodaniel it looks like it's happening to both of us when we migrate, so.. leave it.